packages/server/src/schemas/__tests__/schemas.controllers.spec.ts

```diff
@@ -285,6 +289,7 @@ describe('SchemasController', () => {
             label: 'animal_registry',
             address: '0x87c574FB2DF0EaA2dAf5fc4a8A16dd3Ce39011B1',
             asset_manager_address: '0x3Ba4280217e78a0EaEA612c1502FC2e92A7FE5D7',
+            oracle_address: '0x3Ba4280217e78a0EaEA612c1502FC2e92A7FE5D7',
```

Comment:
Do we have a test for when oracle_address is `0x0...`

Reply:
i will add one